app/controllers/user_controller.rb

```diff
@@ -0,0 +1,3 @@
+class UserController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
method này hoạt động ra sao em?

Reply:
Dạ... method này để đảm bảo người dùng đã đăng nhập. Nếu chưa đăng nhập nó sẽ chuyển hướng trang về login. Đảm bảo việc user đăng nhập nên ta hoàn toàn có thể sử dụng method current_user